src/cluster.cr

```diff
@@ -0,0 +1,20 @@
+class Cluster
+
+    def self.fork (env : Hash)
+        env["FORKED"] = "1"
+        return Process.fork { Process.run(PROGRAM_NAME, nil, env, true, false, true, true, true, nil ) }
+    end
+
+    def self.isMaster
+        (ENV["FORKED"] ||= "0") == "0"
```

Comment:
You shouldn't set value in checker method. Proper way would be `(ENV["FORKED"]? || "0") == "0"`.